css/_details.scss

```diff
@@ -71,6 +71,12 @@
 .contactdetails__header #details-actions {
 	display: flex;
 	flex-shrink: 0;
+	padding: 14px;
+	width: 44px;
+}
+
+.contactdetails__header #details-actions span.icon-more-white {
+	cursor: pointer;
```

Comment:
According to the docs https://docs.nextcloud.com/server/13/developer_manual/design/popovermenu.html#technical-details
![img](https://docs.nextcloud.com/server/13/developer_manual/_images/popover-position.png)
> The required right distance to the border (or padding, whatever you want to use) of the three-dot icon should be 14px (5 for menu margin and 6 for arrow position)

Especially since the click handler is on the icon-more and not in the parent. Which makes the more element too small ;)

|bad|good|
|:--:|:--:|
|![capture d ecran_2018-05-03_10-04-49](https://user-images.githubusercontent.com/14975046/39565755-7e3c5f30-4eb9-11e8-99b7-f3de7f0a731a.png) |![capture d ecran_2018-05-03_10-04-25](https://user-images.githubusercontent.com/14975046/39565757-7f9d8e80-4eb9-11e8-9758-0fff82b21782.png)|



You need to change the span to a div (or put a display:block) and set the `padding:14px`, `height:44px` and `width:44px` to the icon-more div. Otherwise it's not centered :)